app/forms/declaration/CommodityDetails.scala

```diff
@@ -23,7 +23,7 @@ import play.api.data.{Form, Mapping}
 import play.api.libs.json.Json
 import utils.validators.forms.FieldValidator._
 
-case class CommodityDetails(combinedNomenclatureCode: Option[String], descriptionOfGoods: String)
+case class CommodityDetails(combinedNomenclatureCode: Option[String], descriptionOfGoods: Option[String])
```

Comment:
I think I prefer this way of creating optional & required mappings.  The ones I have done so far use different apply/unapply methods (idea borrowed from `TotalPackageQuantity`)